oe_content.module

```diff
@@ -74,3 +75,31 @@ function oe_content_rdf_entity_access(EntityInterface $entity, $operation, Accou
   $provenance_uri = \Drupal::config('oe_content.settings')->get('provenance_uri');
   return $entity->get('provenance_uri')->value === $provenance_uri ? AccessResult::neutral() : AccessResult::forbidden();
 }
+
+/**
+ * Implements hook_entity_load().
+ */
+function oe_content_taxonomy_term_load(array $entities) {
+  // Taxonomy terms have a status field for handling publishing state. We need
+  // to default to TRUE for all terms.
+  foreach ($entities as $entity) {
+    $entity->set('status', TRUE);
+  }
+}
+
+/**
+ * Implements hook_query_QUERY_TAG_alter() for the taxonomy_term_access tag.
+ */
+function oe_content_query_taxonomy_term_access_alter(AlterableInterface $query) {
+  if (!\Drupal::moduleHandler()->moduleExists('rdf_taxonomy')) {
+    return;
+  }
```

Comment:
This is not strictly necessary since this module depends from `rdf_taxonomy`. Maybe a reference to the todo note from the above comment would be enough?